Build broken on Mac after PR #530

When I try to build on my Mac after #530 is merged into develop I get the following error:

```
Building Gregorio; options:
Making all in src
flex -o gabc/gabc-score-determination-l.c --header-file=gabc/gabc-score-determination-l.h gabc/gabc-score-determination.l
flex -o gabc/gabc-notes-determination-l.c gabc/gabc-notes-determination.l
bison -y -d -p "gabc_score_determination_" -o gabc/gabc-score-determination-y.c gabc/gabc-score-determination.y
gabc/gabc-score-determination.y: warning: 3 shift/reduce conflicts [-Wconflicts-sr]
flex -o vowel/vowel-rules-l.c --header-file=vowel/vowel-rules-l.h vowel/vowel-rules.l
bison -y -d -o vowel/vowel-rules-y.c vowel/vowel-rules.y
/Applications/Xcode.app/Contents/Developer/usr/bin/make  all-am
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gregorio-characters.o -MD -MP -MF .deps/gregorio-characters.Tpo -c -o gregorio-characters.o `test -f 'characters.c' || echo './'`characters.c
mv -f .deps/gregorio-characters.Tpo .deps/gregorio-characters.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gregorio-gregorio-utils.o -MD -MP -MF .deps/gregorio-gregorio-utils.Tpo -c -o gregorio-gregorio-utils.o `test -f 'gregorio-utils.c' || echo './'`gregorio-utils.c
mv -f .deps/gregorio-gregorio-utils.Tpo .deps/gregorio-gregorio-utils.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gregorio-messages.o -MD -MP -MF .deps/gregorio-messages.Tpo -c -o gregorio-messages.o `test -f 'messages.c' || echo './'`messages.c
mv -f .deps/gregorio-messages.Tpo .deps/gregorio-messages.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gregorio-struct.o -MD -MP -MF .deps/gregorio-struct.Tpo -c -o gregorio-struct.o `test -f 'struct.c' || echo './'`struct.c
mv -f .deps/gregorio-struct.Tpo .deps/gregorio-struct.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gregorio-unicode.o -MD -MP -MF .deps/gregorio-unicode.Tpo -c -o gregorio-unicode.o `test -f 'unicode.c' || echo './'`unicode.c
mv -f .deps/gregorio-unicode.Tpo .deps/gregorio-unicode.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gregorio-sha1.o -MD -MP -MF .deps/gregorio-sha1.Tpo -c -o gregorio-sha1.o `test -f 'sha1.c' || echo './'`sha1.c
mv -f .deps/gregorio-sha1.Tpo .deps/gregorio-sha1.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT dump/gregorio-dump.o -MD -MP -MF dump/.deps/gregorio-dump.Tpo -c -o dump/gregorio-dump.o `test -f 'dump/dump.c' || echo './'`dump/dump.c
mv -f dump/.deps/gregorio-dump.Tpo dump/.deps/gregorio-dump.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gregoriotex/gregorio-gregoriotex-write.o -MD -MP -MF gregoriotex/.deps/gregorio-gregoriotex-write.Tpo -c -o gregoriotex/gregorio-gregoriotex-write.o `test -f 'gregoriotex/gregoriotex-write.c' || echo './'`gregoriotex/gregoriotex-write.c
mv -f gregoriotex/.deps/gregorio-gregoriotex-write.Tpo gregoriotex/.deps/gregorio-gregoriotex-write.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gregoriotex/gregorio-gregoriotex-position.o -MD -MP -MF gregoriotex/.deps/gregorio-gregoriotex-position.Tpo -c -o gregoriotex/gregorio-gregoriotex-position.o `test -f 'gregoriotex/gregoriotex-position.c' || echo './'`gregoriotex/gregoriotex-position.c
mv -f gregoriotex/.deps/gregorio-gregoriotex-position.Tpo gregoriotex/.deps/gregorio-gregoriotex-position.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gabc/gregorio-gabc-elements-determination.o -MD -MP -MF gabc/.deps/gregorio-gabc-elements-determination.Tpo -c -o gabc/gregorio-gabc-elements-determination.o `test -f 'gabc/gabc-elements-determination.c' || echo './'`gabc/gabc-elements-determination.c
mv -f gabc/.deps/gregorio-gabc-elements-determination.Tpo gabc/.deps/gregorio-gabc-elements-determination.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gabc/gregorio-gabc-write.o -MD -MP -MF gabc/.deps/gregorio-gabc-write.Tpo -c -o gabc/gregorio-gabc-write.o `test -f 'gabc/gabc-write.c' || echo './'`gabc/gabc-write.c
mv -f gabc/.deps/gregorio-gabc-write.Tpo gabc/.deps/gregorio-gabc-write.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gabc/gregorio-gabc-glyphs-determination.o -MD -MP -MF gabc/.deps/gregorio-gabc-glyphs-determination.Tpo -c -o gabc/gregorio-gabc-glyphs-determination.o `test -f 'gabc/gabc-glyphs-determination.c' || echo './'`gabc/gabc-glyphs-determination.c
mv -f gabc/.deps/gregorio-gabc-glyphs-determination.Tpo gabc/.deps/gregorio-gabc-glyphs-determination.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gabc/gregorio-gabc-score-determination-y.o -MD -MP -MF gabc/.deps/gregorio-gabc-score-determination-y.Tpo -c -o gabc/gregorio-gabc-score-determination-y.o `test -f 'gabc/gabc-score-determination-y.c' || echo './'`gabc/gabc-score-determination-y.c
gabc/gabc-score-determination-y.c:2252:3: warning: implicit declaration of
      function 'YYID' is invalid in C99 [-Wimplicit-function-declaration]
  YYLLOC_DEFAULT (yyloc, (yylsp - yylen), yylen);
  ^
gabc/gabc-score-determination.y:44:13: note: expanded from macro
      'YYLLOC_DEFAULT'
        if (YYID (N)) { \
            ^
1 warning generated.
mv -f gabc/.deps/gregorio-gabc-score-determination-y.Tpo gabc/.deps/gregorio-gabc-score-determination-y.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gabc/gregorio-gabc-score-determination-l.o -MD -MP -MF gabc/.deps/gregorio-gabc-score-determination-l.Tpo -c -o gabc/gregorio-gabc-score-determination-l.o `test -f 'gabc/gabc-score-determination-l.c' || echo './'`gabc/gabc-score-determination-l.c
gabc/gabc-score-determination-l.c:2536:17: warning: comparison of integers of
      different signs: 'int' and 'yy_size_t' (aka 'unsigned long')
      [-Wsign-compare]
        for ( i = 0; i < _yybytes_len; ++i )
                     ~ ^ ~~~~~~~~~~~~
1 warning generated.
mv -f gabc/.deps/gregorio-gabc-score-determination-l.Tpo gabc/.deps/gregorio-gabc-score-determination-l.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT gabc/gregorio-gabc-notes-determination-l.o -MD -MP -MF gabc/.deps/gregorio-gabc-notes-determination-l.Tpo -c -o gabc/gregorio-gabc-notes-determination-l.o `test -f 'gabc/gabc-notes-determination-l.c' || echo './'`gabc/gabc-notes-determination-l.c
gabc/gabc-notes-determination-l.c:9297:17: warning: comparison of integers of
      different signs: 'int' and 'yy_size_t' (aka 'unsigned long')
      [-Wsign-compare]
        for ( i = 0; i < _yybytes_len; ++i )
                     ~ ^ ~~~~~~~~~~~~
1 warning generated.
mv -f gabc/.deps/gregorio-gabc-notes-determination-l.Tpo gabc/.deps/gregorio-gabc-notes-determination-l.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT vowel/gregorio-vowel.o -MD -MP -MF vowel/.deps/gregorio-vowel.Tpo -c -o vowel/gregorio-vowel.o `test -f 'vowel/vowel.c' || echo './'`vowel/vowel.c
mv -f vowel/.deps/gregorio-vowel.Tpo vowel/.deps/gregorio-vowel.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT vowel/gregorio-vowel-rules-l.o -MD -MP -MF vowel/.deps/gregorio-vowel-rules-l.Tpo -c -o vowel/gregorio-vowel-rules-l.o `test -f 'vowel/vowel-rules-l.c' || echo './'`vowel/vowel-rules-l.c
vowel/vowel-rules-l.c:1679:17: warning: comparison of integers of different
      signs: 'int' and 'yy_size_t' (aka 'unsigned long') [-Wsign-compare]
        for ( i = 0; i < _yybytes_len; ++i )
                     ~ ^ ~~~~~~~~~~~~
1 warning generated.
mv -f vowel/.deps/gregorio-vowel-rules-l.Tpo vowel/.deps/gregorio-vowel-rules-l.Po
gcc -DHAVE_CONFIG_H -DBRANCH_VERSION='"develo-test-2ad2fcd-2303"' -I.  -I../src -I../src/gabc  -I../src/dump  -I../src/gregoriotex -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2   -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra -MT vowel/gregorio-vowel-rules-y.o -MD -MP -MF vowel/.deps/gregorio-vowel-rules-y.Tpo -c -o vowel/gregorio-vowel-rules-y.o `test -f 'vowel/vowel-rules-y.c' || echo './'`vowel/vowel-rules-y.c
mv -f vowel/.deps/gregorio-vowel-rules-y.Tpo vowel/.deps/gregorio-vowel-rules-y.Po
gcc  -arch x86_64 -g -O2  -Wdeclaration-after-statement -fstack-protector-strong -fPIE -Wformat -Werror=format-security -Wstrict-prototypes -Wall -Wextra  -arch x86_64  -fPIE -pie -o gregorio gregorio-characters.o gregorio-gregorio-utils.o gregorio-messages.o gregorio-struct.o gregorio-unicode.o gregorio-sha1.o dump/gregorio-dump.o gregoriotex/gregorio-gregoriotex-write.o gregoriotex/gregorio-gregoriotex-position.o  gabc/gregorio-gabc-elements-determination.o gabc/gregorio-gabc-write.o gabc/gregorio-gabc-glyphs-determination.o gabc/gregorio-gabc-score-determination-y.o gabc/gregorio-gabc-score-determination-l.o gabc/gregorio-gabc-notes-determination-l.o vowel/gregorio-vowel.o vowel/gregorio-vowel-rules-l.o vowel/gregorio-vowel-rules-y.o  
clang: warning: argument unused during compilation: '-pie'
Undefined symbols for architecture x86_64:
  "_YYID", referenced from:
      _gabc_read_score in gregorio-gabc-score-determination-y.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [gregorio] Error 1
make[1]: *** [all] Error 2
make: *** [all-recursive] Error 1
Failed to build Gregorio.
```

--- a/src/gabc/gabc-score-determination.y
+++ b/src/gabc/gabc-score-determination.y
@@ -40,20 +40,18 @@
 #include "gabc.h"
 
 #define YYLLOC_DEFAULT(Current, Rhs, N) \
-    do { \
-        if (YYID (N)) { \
-            (Current).first_line   = YYRHSLOC(Rhs, 1).first_line; \
-            (Current).first_column = YYRHSLOC (Rhs, 1).first_column; \
-            (Current).first_offset = YYRHSLOC (Rhs, 1).first_offset; \
-            (Current).last_line    = YYRHSLOC (Rhs, N).last_line; \
-            (Current).last_column  = YYRHSLOC (Rhs, N).last_column; \
-            (Current).last_offset  = YYRHSLOC (Rhs, N).last_offset; \
-        } else { \
-            (Current).first_line   = (Current).last_line   = YYRHSLOC (Rhs, 0).last_line; \
-            (Current).first_column = (Current).last_column = YYRHSLOC (Rhs, 0).last_column; \
-            (Current).first_offset = (Current).last_offset = YYRHSLOC (Rhs, 0).last_offset; \
-        } \
-    } while (YYID (0))
+    if (N) { \
+        (Current).first_line   = YYRHSLOC(Rhs, 1).first_line; \
+        (Current).first_column = YYRHSLOC (Rhs, 1).first_column; \
+        (Current).first_offset = YYRHSLOC (Rhs, 1).first_offset; \
+        (Current).last_line    = YYRHSLOC (Rhs, N).last_line; \
+        (Current).last_column  = YYRHSLOC (Rhs, N).last_column; \
+        (Current).last_offset  = YYRHSLOC (Rhs, N).last_offset; \
+    } else { \
+        (Current).first_line   = (Current).last_line   = YYRHSLOC (Rhs, 0).last_line; \
+        (Current).first_column = (Current).last_column = YYRHSLOC (Rhs, 0).last_column; \
+        (Current).first_offset = (Current).last_offset = YYRHSLOC (Rhs, 0).last_offset; \
+    }
 
 #include "gabc-score-determination.h"
 #include "gabc-score-determination-l.h"

